src/utils/isValidTime.ts

```diff
@@ -0,0 +1,7 @@
+function isValidTime(hour, minute) {
+    if (hour < 0 || hour > 23 || minute < 0 || minute > 59) return false;
+
+    return true;
+}
```

Comment:
Seems like a good use case for ternary operator :Java:

Reply:
Good idea

Reply:
Lol now that I look at this, you don't need ternary at all, you can just do a return since it's a boolean 🤣

Reply:
I'm gonna stick with this, otherwise it'll be too hard to read with `return !()`